README.md

```diff
@@ -104,7 +104,7 @@ Contributions written in other langauges will likely be rejected.
 |         | rm        | Remove files or directories                      |
 |         | rmdir     | Remove empty directories                         |
 |         | runcon    | Run a command in specified SELinux context       |
-|         | seq       | Print numeric sequences                          |
+| &check; | seq       | Print numeric sequences                          |
```

Comment:
Please change the `Completed (x/109)` number at the top of the table, as well.

Reply:
thanks! missed again :sweat_smile: